### ISSUE TITLE
Leaving full screen mode tricky

### ISSUE THREAD

**Issue by ghost** [kloener/visor-android#8]
Because people may not realise the app switched the device to landscape mode they will try to swipe down on the wrong side.

Perhaps there could be a few options offered in the "i" menu as well? Or yet another button?

**Comment by kloener** [comment]
You're right! It's not obvious where to escape the fullscreen mode... I'm not a friend of yet another button, but a settings/options menu would be useful to enable/disable the immersive mode. Furthermore, it can be used to change camera's resolution 🤔 ...

**Comment by ghost** [comment]
Maybe plain fullscreen mode would be good enough?
